services/question-service/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
+    ports:
+      - "8080:8080"
+    depends_on:
+      - mongodb
+    links:
+      - mongodb
+    environment:
+      - MONGO_URI=mongodb://mongodb:27017/question-service-api
+  mongodb:
+    image: mongo:6.0
+    volumes:
+      - ./data:/data/db
+    ports:
+      - "27017:27017"
+    
```

Comment:
Please bring/appened over the whole docker-compose configs to the main docker-compose.yml file at `Server-Configs` folder

Reply:
@Acerizm I will open another pull request for this.